ting_search.module

```diff
@@ -973,5 +973,8 @@ function ting_search_submit($form, &$form_state) {
  */
 function _ting_search_quote($string) {
   $cqlDoc = new TingSearchCqlDoctor($string);
-  return $cqlDoc->string_to_cql($string);
+  // Hold cql query for further changes.
+  $cql_query = $cqlDoc->string_to_cql();  
+  // Return cql query.
+  return $cql_query;
```

Comment:
As mentioned in ding2/ting#42 ting_search should not be moved. Consequently there are no longer any relevant changes in this pull request.